docs/faq.rst

```diff
@@ -69,9 +69,11 @@ Here's an example spider using BeautifulSoup API, with ``lxml`` as the HTML pars
 What Python versions does Scrapy support?
 -----------------------------------------
 
-Scrapy is supported under Python 2.7 and Python 3.4+.
+Scrapy is supported under Python 2.7 and Python 3.4+
+under CPython (default Python implementation) and PyPy (only for Python 2.7).
```

Comment:
Do you know what's the reason pypy3 is not supported? Is it just us not running tests for it, or are there things which work in pypy2, but not in pypy3?

Reply:
Last time I checked, lxml did not install on PyPy 3. I filed an issue about it and it was quickly fixed, but then it stopped working again. I didn't check in a while though, let me check again.